src/main/java/xyz/nucleoid/plasmid/game/GameWorld.java

```diff
@@ -94,33 +130,65 @@ public void setGame(Game game) {
         this.invoker(GameOpenListener.EVENT).onOpen();
     }
 
+    /**
+     * Closes this {@link GameWorld}.
```

Comment:
This closes specifically the current Game instance stored and notifies it, but doesn't close the world itself

Reply:
Fixed. Any thoughts on whether docs should tell the caller about notifying listeners (eg. `"All XYZ listeners are notified of this action."`)?

Reply:
Oh, yeah. That would be nice to document. :)

Reply:
Sounds good, just wanted to make sure I should leave it in. All methods that call listeners in `GameWorld` mention it in one way or another now, and I'll continue that when I document other classes.